wwwroot/test/csv/regionMapping.json

```diff
@@ -0,0 +1,197 @@
+{
+    "comments": "Matching takes place in the order defined in this file. Place code matches before name matches, and smaller regions before larger ones.",
```

Comment:
As you probably know, JSON keys are unordered.  So if this currently works in all browsers it's just a coincidence.  Unless you're talking about the order in the `aliases` array, in which case maybe clarify this in the comment?